Mathlib/CategoryTheory/Galois/Topology.lean

```diff
@@ -109,6 +108,75 @@ instance (X : C) : ContinuousSMul (Aut (F.obj X)) (F.obj X) := by
   constructor
   fun_prop
 
+instance continuousSMul_aut_fiber (X : C) : ContinuousSMul (Aut F) (F.obj X) where
+  continuous_smul := by
+    let g : Aut (F.obj X) × F.obj X → F.obj X := fun ⟨σ, x⟩ ↦ σ.hom x
+    let h (q : Aut F × F.obj X) : Aut (F.obj X) × F.obj X :=
+      ⟨((fun p ↦ p X) ∘ autEmbedding F) q.1, q.2⟩
+    show Continuous (g ∘ h)
+    fun_prop
+
+variable [GaloisCategory C] [FiberFunctor F]
+
+/--
+If `H` is an open subset of `Aut F` such that `1 ∈ H`, there exists a finite
+set `I` of connected objects of `C` such that every `σ : Aut F` that induces the identity
+on `F.obj X` for all `X ∈ I` is contained in `H`. In other words: The kernel
+of the evaluation map `Aut F →* ∏ X : I ↦ Aut (F.obj X)` is contained in `H`.
+-/
+lemma exists_set_ker_evaluation_subset_of_isOpen {H : Set (Aut F)} (hone : 1 ∈ H)
+    (h : IsOpen H) : ∃ (I : Set C) (_ : Fintype I), (∀ X ∈ I, IsConnected X) ∧
+    (∀ σ : Aut F, (∀ X : I, σ.hom.app X = 𝟙 (F.obj X)) → σ ∈ H) := by
```

Comment:
To slightly increase readability, I suggest starting the conclusion on a new line
```suggestion
lemma exists_set_ker_evaluation_subset_of_isOpen
    {H : Set (Aut F)} (hone : 1 ∈ H) (h : IsOpen H) :
    ∃ (I : Set C) (_ : Fintype I), (∀ X ∈ I, IsConnected X) ∧
      (∀ σ : Aut F, (∀ X : I, σ.hom.app X = 𝟙 (F.obj X)) → σ ∈ H) := by
```
Minor nitpick: I slightly prefer `h1` over `hone`, since "hone" is also a verb.